ONSdigital/dp-frontend-articles-controller
# Feature/census branding

### What

Adds Census 2021 branding to the status header and additional "About the data" section when the census survey flag is present on the article.

<img width="992" alt="Screenshot 2022-09-16 at 15 24 43" src="https://user-images.githubusercontent.com/912770/190663551-8a2399eb-cfe0-45cf-ba92-791a7fb14a4f.png">
<img width="983" alt="Screenshot 2022-09-16 at 15 29 14" src="https://user-images.githubusercontent.com/912770/190663563-421ed7f0-9fb2-49b6-90db-dc387b949c94.png">

Please note that the phrase "This release includes data from Census 2021" is subject to change and is present for now.

### How to review

- In a separate terminal, run dp-design-system with `make debug`
- Run this frontend with `make debug`
- Visit an article with Census branding, for example http://localhost:26500/employmentandlabourmarket/peopleinwork/labourproductivity/articles/introducingquarterlyregionallabourinputmetrics/july2022
- Verify that the Census branding is present in English and Welsh

### Who can review

Go / Design System frontend developers
